stacks/spica/src/bucket/pages/index/index.component.spec.ts

```diff
@@ -25,10 +25,12 @@ import {ActivatedRoute} from "@angular/router";
 import {RouterTestingModule} from "@angular/router/testing";
```

Comment:
I had made these changes because of an issue with the native table in the material.
See: https://github.com/angular/components/issues/14375